cmd/orb.go

```diff
@@ -1297,13 +1309,24 @@ func initOrb(opts orbOptions) error {
 		return y[0]
 	}()
 
-	circleConfig, err := ioutil.ReadFile(path.Join(orbPath, ".circleci", "config.yml"))
+	circleConfig1, err := ioutil.ReadFile(path.Join(orbPath, ".circleci", "config.yml"))
 	if err != nil {
 		return err
 	}
 
-	circle := string(circleConfig)
-	err = ioutil.WriteFile(path.Join(orbPath, ".circleci", "config.yml"), []byte(orbTemplate(circle, projectName, ownerName, orbName, namespace)), 0644)
+	config1String := string(circleConfig1)
```

Comment:
Is `config1string` and `config2string` still needed?

Reply:
If I understand correctly, these are both my new changes to this orb. We now use two config files rather than one as we did previously, as orb tools 11 uses dynamic config.